index.html

```diff
@@ -76,7 +76,7 @@ <h2>Parsers</h2>
     <div class="modules skills side-margin">
       <h2>Skills</h2>
       <p>
-        Skills are what makes opsdroid tick. They define how opsdroid should respond and what actions to take.
+        Skills are what makes Opsdroid tick. They define how Opsdroid should respond and what actions to take.
```

Comment:
opsdroid should be lowercase - this is mostly due to the fact that we are using lowercase everywhere else 👍